apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClient.cs

```diff
@@ -994,6 +998,7 @@ public BigtableClientImpl(Bigtable.BigtableClient grpcClient, BigtableSettings s
         partial void Modify_SampleRowKeysRequest(ref SampleRowKeysRequest request, ref CallSettings settings);
         partial void Modify_MutateRowRequest(ref MutateRowRequest request, ref CallSettings settings);
         partial void Modify_MutateRowsRequest(ref MutateRowsRequest request, ref CallSettings settings);
+        partial void Modify_MutateRowsResponse(MutateRowsRequest request, ref MutateRowsStream response, CallSettings originalCallSettings, CallSettings settings);
```

Comment:
So would you anticipate this being generated for all streaming RPCs, but only those?